src/components/IssueFieldState.vue

```diff
@@ -44,15 +44,15 @@ function onPromote(flawId: string) {
 }
 
 function nextPhase(flawStatus: WorkflowPhases) {
-  const [labels, phases] = [Object.keys(workflowStatuses), Object.values(workflowStatuses)];
+  const [labels, phases] = [Object.keys(workflowStates), Object.values(workflowStates)];
   const index = phases.indexOf(flawStatus);
   return labels[index + 1] || labels.slice(-1)[0];
```

Comment:
Should we change `flawStatus` parameter to `flawState`?

Reply:
Agree for changing `flawStatus` to `flawState`. Otherwise LGTM ✔️